API/appsettings.Development.json

```diff
@@ -3,6 +3,11 @@
     "DefaultConnection": "Data source=kavita.db"
   },
   "TokenKey": "super secret unguessable key",
+  "StatsOptions": {
```

Comment:
As discussed, move this into the DB. Url and secret can be hardcoded in the code.